server/package.json

```diff
@@ -21,6 +21,7 @@
     "mongoose": "^4.8.5",
     "passport": "^0.3.2",
     "passport-github2": "^0.1.10",
+    "redis": "^2.7.1",
```

Comment:
What's up with the version? Just curious if there's a reason. 
![image](https://cloud.githubusercontent.com/assets/18563015/24055055/fb1a87d2-0b14-11e7-9651-490c920379f1.png)